Services/CheckRightsService/Repositories/CheckRightsRepository.cs

```diff
@@ -19,5 +21,21 @@ public List<DbRight> GetRightsList()
         {
             return dbContext.Rights.ToList();
         }
+
+        public bool CheckIfUserHasRight(Guid userId, int rightId)
+        {
+            var rights = dbContext.Rights
+                .AsNoTracking()
+                .Include(r => r.UserIds);
+            if (rights
+                .Any(r => r.UserIds
+                    .Select(ru => ru.UserId)
+                        .Contains(userId)))
```

Comment:
Fix spaces.

Reply:
Jesus you won't let me close this will ya.
Should this be fixed? It's crystal clear to read, @lalagvana  liked it, I like it.
This is subjective. Anyway we agreed to indent LINQ methods, didn't we?

Reply:
Enjoy. Hope you like this version much more.